### PR TITLE
Configure CI/CD

### PR DESCRIPTION
**Pull request by euggeze**
Created a build configuration. Added coveralls and pylint to the configuration. Added badges to readme file.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[run]
+omit =
+	setup.py
+	department_project/manage.py
+	*/tests*
+	*wsgi.py
+    	*asgi.py
+	*admin.py
+	*apps.py
```
Comment (inauros) [inline .coveragerc:9]: Please could you explain, why you ignored these files?

File: .github/workflows/python_package.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: python_package
+
+on: [push]
```
Comment (inauros) [inline .github/workflows/python_package.yml]: Please include pull requests as well.

File: .github/workflows/python_package.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint
+          pip install pylint-django
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```
Comment (inauros) [inline .github/workflows/python_package.yml]: Please remove the "if" statement. The build should fail if the requirements.txt file does not exist.

File: .gitignore [hunk 4]
```diff
@@ -6,3 +6,5 @@ pyvenv.cfg
 Department_application.egg-info/
 dist/
 **/__pycache__
+.coveralls.yml
```
Comment (inauros) [inline .gitignore:9]: Why this file was added to .gitignore?

File: .pylintrc [hunk 5]
```diff
@@ -0,0 +1,626 @@
...
+        next-method-defined,
+        dict-items-not-iterating,
+        dict-keys-not-iterating,
+        dict-values-not-iterating,
+        deprecated-operator-function,
+        deprecated-urllib-function,
+        xreadlines-attribute,
+        deprecated-sys-function,
+        exception-escape,
+        comprehension-escape
```
Comment (inauros) [inline .pylintrc]: Please clear this list. Errors that should really be ignored will be added at development time step by step.

File: .pylintrc [hunk 6]
```diff
@@ -0,0 +1,626 @@
...
+
+# Number of spaces of indent required inside a hanging or continued line.
+indent-after-paren=4
+
+# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
+# tab).
+indent-string='    '
+
+# Maximum number of characters on a single line.
+max-line-length=100
```
Comment (inauros) [inline .pylintrc]: please replace 100 with 120 - we use exactly this line length

**Review by inauros — CHANGES_REQUESTED** [review]
Please fix the following errors generated by pylint:
************* Module department_app.tests.placeholder_test
department_project/department_app/tests/placeholder_test.py:2:0: W0311: Bad indentation. Found 1 spaces, expected 4 (bad-indentation) 
department_project/department_app/tests/placeholder_test.py:1:0: C0114: Missing module docstring (missing-module-docstring) 
department_project/department_app/tests/placeholder_test.py:1:0: C0116: Missing function or method docstring (missing-function-docstring) 
**Fixed**

File: .coveragerc (hunk 1 above)
Comment (euggeze) [reply .coveragerc:9]: These files are not included in the program.

File: .github/workflows/python_package.yml (hunk 2 above)
Comment (euggeze) [reply .github/workflows/python_package.yml]: Done

File: .github/workflows/python_package.yml (hunk 3 above)
Comment (euggeze) [reply .github/workflows/python_package.yml]: Fixed

File: .gitignore (hunk 4 above)
Comment (euggeze) [reply .gitignore:9]: There is a token in this file, which must not be public.

File: .pylintrc (hunk 5 above)
Comment (euggeze) [reply .pylintrc]: Done

File: .pylintrc (hunk 6 above)
Comment (euggeze) [reply .pylintrc]: Done

**Review by inauros — APPROVED** [review]

**Pull request merged**